test/specs/test.e2e.ts

```diff
@@ -0,0 +1,13 @@
+import { $, expect } from '@wdio/globals'
+
+describe('Electron Testing', () => {
+    it('should display loader on start', async () => {
+        const header = $('.loader')
```

Comment:
boo, this didn't work. :( we need to get the window id and make sure we are using the proper window: https://stackoverflow.com/questions/75462881/message-no-such-window-target-window-already-closed